config/services/llmnr.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <service>
   <short>LLMNR</short>
-  <description>Link-Local Multicast Name Resolution (LLMNR) allows both IPv4 and IPv6 hosts to perform name resolution for hosts on the same local link.</description>
+  <description>Link-Local Multicast Name Resolution (LLMNR) allows both IPv4 and IPv6 hosts to perform name resolution for hosts on the same local link. This service matches incoming queries; it will allow this host to be resolved by other hosts.</description>
   <include service="llmnr-tcp"/>
   <include service="llmnr-udp"/>
```

Comment:
What do you think about allowing `llmnr-client` here? I think a user enabling `llmnr` will also want to hear the replies.

Note that `ws-discovery` does NOT include `ws-discovery-client`, but maybe it should. I wonder how @gbsf and @marcosfrm feel about this.

Reply:
We should avoid `source-port` as much as possible. rfc4795 says implementations are free to discard unsolicited responses...

Reply:
@marcosfrm, fair enough. Let's leave out the `-client` variants then.

Reply:
The thing is, while the `-client` services are a kludge, they are basically required to use on desktop machines. Until we can get conntrackd working, we have to rely on an implementation detail that Linux applications in general use the same request port number as the response, instead of a random one. The multicast-to-unicast dance leaves no other option, otherwise we end up discarding _solicited_ responses too…
If they are not enabled by default alongside the main service, I feel that at least the description of those should mention that `-client` is currently needed to receive replies.

Reply:
Let me ask it this way.

Would a user _expect_ that enabling the `llmnr` service will allow them to make queries?

Reply:
I stand against any rule with `source-port` being enabled indirectly.